script.js

```diff
@@ -0,0 +1,225 @@
+let menu = document.querySelector(".menu");
+let humanVhumanBtn = document.querySelector(".humanVhuman");
+let humanVcpuBtn = document.querySelector(".humanVcpu");
+let gameBoardContainer = document.querySelector(".gameBoardContainer");
+let markX = document.querySelector(".markX");
+let markO = document.querySelector(".markO");
+let turnDecider = document.querySelector(".turnDecider");
+let XscoresDisplay = document.querySelector(".XscoresDisplay");
+let OscoresDisplay = document.querySelector(".OscoresDisplay");
+let TieScoresDisplay = document.querySelector(".TieScoresDisplay");
+let nextRoundBtn = document.querySelectorAll(".nextRound")
+let refree = document.querySelectorAll(".refree")
+let restartBtn = document.querySelector(".restartBtn")
+let quitBtn = document.querySelectorAll(".quit")
+nextRoundBtn.forEach(btn => {
+    btn.addEventListener("click",()=>{
+        resetGameData()
+        refree.forEach(element => {
+            element.style.display = "none"
+        });
+    });
+});
+restartBtn.addEventListener("click",()=>{
+    resetGameData()
+    refree.forEach(element => {
+        element.style.display = "none"
+    });
+})
+quitBtn.forEach(btn => {
+    btn.addEventListener("click",()=>{
+        location.reload()
+    })
+});
+humanVhumanBtn.addEventListener("click",()=>{
+    userData.setMode(0)
+    menu.style.display = "none"
+    gameBoardContainer.style.display = "flex"
+})
+humanVcpuBtn.addEventListener("click",()=>{
+    userData.setMode(1)
+    menu.style.display = "none"
+    gameBoardContainer.style.display = "flex"
+})
+let userData = (function(){
+    let _mode = 0
+    let _gameBoard = [0,1,2,3,4,5,6,7,8]
+    let markX = "assets2/x.svg"
+    let markO = "assets2/o.svg"
+    let turnDecider_O = "assets2/oturn.svg"
+    let turnDecider_X = "assets2/xturn.svg"
+    let _winner = false
+    let _draw = false
+    let _gameover = false
+    let _drawCount = 0
+    let _player1Score = 0
+    let _player2Score = 0
+    let _tieScores = 0
+    let _winningConditions = [
```

Comment:
The underscores are to indicate that they are private variables and can't be altered from anywhere else because they are'nt getting returned.
so for instance to change player1Score, i would have to use the designated methods.